include/IrrCompileConfig.h

```diff
@@ -132,6 +132,10 @@
 #endif
 #endif
 
+#if defined(__FreeBSD__) || defined(__OpenBSD__)
+#define NO_IRR_COMPILE_WITH_JOYSTICK_EVENTS_
+#endif

```

Comment:
Can better be put as an ifdef around this line:
![grafik](https://user-images.githubusercontent.com/1042418/131216326-aae916be-543b-4f47-8742-ad5ddde83d52.png)

Reply:
I tried something like that, but I may have to undo it because I should've asked for clarification first. I'm confused about using an ifdef because it would basically need an ifndef, and I'm not sure that can or should replace defined(), based on the rest of the code which I tried to match the structure of as closely as possible.

Reply:
The way it is now is correct, not sure what the question is.